src/providers/dart_commands.ts

```diff
@@ -0,0 +1,47 @@
+"use strict";
+
+import * as as from "../analysis/analysis_server_types";
+import * as editors from "../editors";
+import * as vs from "vscode";
+import { Analyzer } from "../analysis/analyzer";
+
+export class DartCommands  {
+	private analyzer: Analyzer;
+	constructor(context: vs.ExtensionContext, analyzer: Analyzer) {
+		this.analyzer = analyzer;
+
+        context.subscriptions.push(
```

Comment:
I didn't think it does, but they do look like tabs (in GH at least) now. You can always force it be clicking in the status bar where it tells you what the indenting is (which then puts the options up in the command palette which is a bit weird!).